### DISCUSSION TITLE
How to covert a transparent image to solid

### DISCUSSION THREAD

**Discussion by bill-lancaster** [ImageMagick/ImageMagick#2092] [Help] [answered]
I have an image I wish to use as a virtual background for Zoom, it looks rather weird (I thing it's transparent), Zoom says it wants a solid image.
Is there a way to convert the .png image to solid?

**Comment by snibgo** [comment]
What version of IM, on what platform?

How about:

magick in.png -alpha off out.png

**Comment by bill-lancaster** [comment, marked as answer]
Sorry about this, I should have checked further.  In fact the Zoom stock virtual background images also have the same problem.
For the record, I'm using Kubuntu 18.10; ImageMagick 6.9.7-4 and 
`: $  convert ~/Pictures/in.png -alpha off ~/out.png`
Ran without errors

**Comment by fmw42** [comment]
You can also flatten the image against any color you want. 

`convert in.png -background white -flatten out.png`
